test/algorithms/set_operations/intersection/intersection_linear_linear.cpp

```diff
@@ -1368,7 +1368,7 @@ BOOST_AUTO_TEST_CASE( test_intersection_ml_ml_degenerate )
                       (9.98265 0.00543606),(9.09826 -100.515944),\
                       (7.08745 -329.0674155),(5.06428 -559.024344),\
                       (3.23365 -767.0972558),(3.16036 -775.427199),\
-                      (-0.7654 8.88178e-16,-0.7654 0,5 3))"),
```

Comment:
So we don't test these degenerate cases anymore?
I'm not against it, but I wonder why - it breaks the new code? Why?

Reply:
This is the result of the intersection (the inputs are a few lines above and they are not changed). So the results here are a bit different with the new code. Actually, they look better since they create multilinestring without the almost duplicate points (-0.7654 8.88178e-16) and (-0.7654 0) that is created since (-0.7654 0) is contained in both input multilinstrings.

Reply:
Okay, that looks fine to me. Thanks!